modules/misc/doc/content/source/postprocessors/ThermocoupleSensorPostprocessor.md

```diff
@@ -0,0 +1,38 @@
+# ThermocoupleSensorPostprocessor
+
+!syntax description /Postprocessors/ThermocoupleSensorPostprocessor
+
+## Description
+
+`ThermocoupleSensorPostprocessor` object implements a thermocouple sensor that takes in the calculated input signal from moose and outputs a realistic and sensor-mediated output that accounts for efficiency, drift, delay and noise of the sensor in question. Different types of thermocouples are included. This is a child class of `GeneralSensorPostprocessor`.
+
+From the `GeneralSensorPostprocessor`,
+\begin{equation}
+ \tilde{u}(t) =  u(t) + f_{SN}(t)n(\sigma(t)) 
+\end{equation}
+
```

Comment:
J E and K type differences should be documented

Reply:
because we want the user to be able to define all variables, we eliminated types of thermocouples all together.